jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -138,3 +143,22 @@ def _get_yaml(template: str, params: Dict, custom_resource_dir: Optional[str] =
         for k, v in params.items():
             content = content.replace(f'{{{k}}}', str(v))
     return content
+
+
+def _patch_yaml(template: str, params: Dict, custom_resource_dir: Optional[str] = None):
+    import yaml
+
+    if custom_resource_dir:
```

Comment:
the custom resource dir logic should not be handled in this function. Instead, the function should just know about the `resource_dir`

Reply:
you're correct